static/src/js/components/Table/Table.jsx

```diff
@@ -80,23 +89,60 @@ const Table = ({
   }
 
   return (
-    <BootstrapTable striped>
-      <thead>
-        <tr>
-          {renderHeaders}
-        </tr>
-      </thead>
-      <tbody>
-        {content}
-      </tbody>
-    </BootstrapTable>
+    <Container>
+      <Row>
+        <Col>
+          <span className="d-block mb-3">
+            Showing
+            {' '}
+            {count > 0 && offset}
+            -
+            {data.length === limit ? offset + limit : offset + data.length}
+            {' '}
+            of
+            {' '}
+            {count}
+            {' '}
+            Results
+          </span>
+        </Col>
+        <Col xs="auto">
+          <div className="mx-auto">
+            {
+              hasPages && (
+                <PaginationComponent
+                  setoffset={setOffset}
+                  limit={limit}
+                  count={count}
+                />
+              )
+            }
+          </div>
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          <BootstrapTable striped>
+            <thead>
+              <tr>
+                {renderHeaders}
+              </tr>
+            </thead>
+            <tbody>
+              {content}
+            </tbody>
+          </BootstrapTable>
+        </Col>
+      </Row>
+    </Container>
   )
 }
 
 Table.defaultProps = {
   classNames: [],
   loading: null,
-  error: 'No Data to Display'
+  error: 'No Data to Display',
```

Comment:
I wonder if we should have a dedicated prop to set an empty state so we can treat no data vs. errors differently